core/src/test/java/com/google/android/fhir/db/impl/dao/LocalChangeUtilsTest.kt

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.db.impl.dao
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.android.fhir.db.impl.dao.LocalChangeUtils.mergeLocalChanges
+import com.google.android.fhir.db.impl.entities.LocalChangeEntity
+import com.google.android.fhir.toTimeZoneString
+import java.util.Date
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+/** Unit tests for [LocalChangeUtils]. */
+@RunWith(RobolectricTestRunner::class)
+class LocalChangeUtilsTest {
+
+  @Test
+  fun mergeLocalChangesTest() {
+    val payload1 =
+      """{
+          "resourceType": "Patient",
+          "id": "human",
+          "name": [
+                  {
+                    "use": "usual",
+                     "given": [
+                     "Kenzi"
+                    ]
+                    }
+           ]}"""
```

Comment:
I would probably do the indentation this way:

```suggestion
      """
      {
        "resourceType": "Patient",
        "id": "human",
        "name": [
          {
            "use": "usual",
            "given": [
              "Kenzi"
            ]
          }
        ]
      }
      """
```